gremlin-python/src/main/jython/gremlin_python/driver/tornado/transport.py

```diff
@@ -26,21 +26,25 @@
 
 class TornadoTransport(AbstractBaseTransport):
 
+    _default_max_content_length = 10 * 1024 * 1024
```

Comment:
This is probably a breaking change and should go either in 350 or in 3411 with same default as current version. In absence of explicitly setting this value, the default must be coming from the websocket framework. 

This is breaking because let's say that the current default is 20MB and a user of the client is relying on that but when we introduce a new default value of 10MB, after upgrade, their queries will suddenly start failing.

Reply:
it was set to precisely the same default as existed before. isn't that correct @krlawrence ?

Reply:
The Tornado default is 10MB The Tornado code includes that exact calculation (10 * 1024 * 1024) in fact. This should not be a breaking change to anyone. I did a lot of testing of this before I opened the original Jira to prove that that is the current limit.

Reply:
Thank you for clarifying. If the default is same as before then this is definitely not breaking. That answers my question.